packages/db/src/runtime/index.ts

```diff
@@ -130,3 +130,17 @@ function handleSerializedSQL<T>(def: T | SerializedSQL) {
 	}
 	return def;
 }
+
+export function normalizeDatabaseUrl(envDbUrl: string | undefined, defaultDbUrl: string): string {
+	if(envDbUrl) {
+		// This could be a file URL, or more likely a root-relative file path.
+		// Convert it to a file URL.
+		if(envDbUrl.startsWith('file://')) {
+			return envDbUrl;
+		}
+		return new URL(envDbUrl, process.cwd()).toString();
```

Comment:
oh, you're right. I was trying to avoid having teh runtime depend on Node builtin modules. Couldn't I do `'file://' + process.cwd()` instead?

I'll also check why the test didn't catch this mistake...

Reply:
Updated.